lib/web/apiserver_test.go

```diff
@@ -2524,6 +2524,45 @@ func TestClusterKubesGet(t *testing.T) {
 	}, resp.Items[0])
 }
 
+func TestClusterDesktopsGet(t *testing.T) {
+	env := newWebPack(t, 1)
+
+	proxy := env.proxies[0]
+	pack := proxy.authPack(t, "test-user@example.com", nil /* roles */)
+
+	type testResponse struct {
+		Items      []ui.Desktop `json:"items"`
+		TotalCount int          `json:"totalCount"`
+	}
+
+	resource, err := types.NewWindowsDesktopV3("desktop1", map[string]string{"test-field": "test-value"}, types.WindowsDesktopSpecV3{
+		Addr:   "addr:3389", // test stripping off rdp port
+		HostID: "host",
+	})
+	require.NoError(t, err)
+
+	// Register a windows desktop
```

Comment:
Test with >1 desktop?

Same for others, if applicable.

Reply:
got it, will add the tests and then include them in master in a subsequent pr